### PR TITLE
Match manage shipments count calculation to transfer agreements

### PR DESCRIPTION
**Pull request by fhenrich33**
Screen Shot 2024-12-10 at 04 51 52

Screen Shot 2024-12-10 at 04 51 55

### CODE REVIEW

**Comment by fhenrich33** [comment]
[2.0 FE Design Bug: Make Display of Active Shipments Count Consistent Between Manage Agreements and Manage Shipments](https://trello.com/c/3JU1gKVk/1637-20-fe-design-bug-make-display-of-active-shipments-count-consistent-between-manage-agreements-and-manage-shipments)

**Comment by fhenrich33** [comment]
> Hey, from the screenshots it looks correct, though I haven't done a functional test of it yet.
> 
> However, I don't see a test written to check the amount? @fhenrich33

@aerinsol I've added an assertion to cover the count number in one of the tabs in the Manage Shipments. This covers that side at least. However, to make a more comprehensive test where we check both Manage Shipments and Transfer Agreements, while it's possible with RLT, it's a bit janky. I'll add this test to the Playwright suite.

**Comment by aerinsol** [comment]
> > Hey, from the screenshots it looks correct, though I haven't done a functional test of it yet.
> > However, I don't see a test written to check the amount? @fhenrich33
> 
> @aerinsol I've added an assertion to cover the count number in one of the tabs in the Manage Shipments. This covers that side at least. However, to make a more comprehensive test where we check both Manage Shipments and Transfer Agreements, while it's possible with RLT, it's a bit janky. I'll add this test to the Playwright suite.

OK great. I'm not familiar with Playwright, so not sure what tests there are supposed to look like - I'd appreciate it if you give me a heads up to let me know when that happens. I'll also comment this in Trello.

**Review by aerinsol — APPROVED** [review]

File: front/src/views/Transfers/ShipmentsOverview/ShipmentsOverview.test.tsx [hunk 1]
```diff
@@ -66,6 +66,8 @@ it("4.4.1.4 - Initial Load of Page", async () => {
   user.click(screen.getByText(/Sending \(/i));
   // Data of Mock Transfer is shown correctly
   expect(await screen.findByRole("cell", { name: /thessaloniki boxcare/i })).toBeInTheDocument();
+  // Display the filtered count for "Sending" shipments.
+  expect(screen.getByText(/Sending \(1\)/i)).toBeInTheDocument();
```
[the remark this answers is not in the document]
Comment (aerinsol) [reply front/src/views/Transfers/ShipmentsOverview/ShipmentsOverview.test.tsx:70]: yeah I see it. FYI for the more extensive test, I definitely want to be able to have several computed shipments (i.e., more than 1) so we can check thoroughly). One of the groupings should also have a zero sum as well. Not needed for this PR.

**Pull request merged**